app/views/scrum/sprints/_table.html.erb

```diff
@@ -2,29 +2,40 @@
   <thead>
```

Comment:
This view is effectively the Google spreadsheet I'm aiming to automate. This is and the chart.